### PR TITLE
port_names argument support for result.PortRequest

### PR DESCRIPTION
**Pull request by alakendu**
This pull request for issue #122

### CODE REVIEW

File: ixnetwork_open_traffic_generator/vport.py [hunk 1]
```diff
@@ -615,8 +615,19 @@ def results(self, request):
             self._column_names = []
         else:
             self._column_names = request.column_names
+        
+        port_filter = {'property': 'name', 'regex': '.*'}
+        port_names = [port.name for port in self._api._config.ports]
+        if request is not None and request.port_names is not None \
```
Comment (ashutshkumr) [inline ixnetwork_open_traffic_generator/vport.py:621]: If we want to be consistent with explicit checks, we should do:
```python
if request is not None and request.port_names is not None and len(request.port_names) > 0:
```

Or if we prefer something terse and implicit, we should do:
```python
if request and request.port_names:
```

File: ixnetwork_open_traffic_generator/vport.py [hunk 2]
```diff
@@ -615,8 +615,19 @@ def results(self, request):
...
+                and request.port_names:
+            port_names = request.port_names
+        if len(port_names) == 1:
+            port_filter['regex'] = '^%s$' % port_names[0]
+        elif len(port_names) > 1:
+            port_filter['regex'] = '^(%s)$' % '|'.join(port_names)
+
```
Comment (ashutshkumr) [inline ixnetwork_open_traffic_generator/vport.py:628]: Shouldn't just executing the snippet under `else` (for regex) work for all cases ?

**Review by ashutshkumr — APPROVED** [review]
Apart from comments on style, LGTM.

**Review by ajbalogh — APPROVED** [review]

File: ixnetwork_open_traffic_generator/vport.py (hunk 1 above)
Comment (alakendu) [reply ixnetwork_open_traffic_generator/vport.py:621]: Actually we coded that way though I don't have any preference. 
Note: Zen of Python said "Explicit is better than implicit".  :)

File: ixnetwork_open_traffic_generator/vport.py (hunk 2 above)
Comment (alakendu) [reply ixnetwork_open_traffic_generator/vport.py:628]: we will ignore corner case when len(port_names) == 0. So I still vote for elif.

File: ixnetwork_open_traffic_generator/vport.py (hunk 1 above)
Comment (alakendu) [reply ixnetwork_open_traffic_generator/vport.py]: add implicit check

**Pull request merged**